components/jupyter-web-app/kubeflow_jupyter/default/templates/notebooks.html

```diff
@@ -1,80 +1,98 @@
 {% extends "layout.html" %}
 {% block javascript %}
-  {{ super() }}
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js"></script>
-  <script src="{{ prefix + url_for('static', filename='js/notebooks.js') }}"></script>
+	{{ super() }}
+    <script defer>
+        // Establish communication with Central Dashboard
+        window.addEventListener('DOMContentLoaded', function (event) {
```

Comment:
Since we aren't running this JS code through any type of transpilation, do you want to keep it as vanilla as possible and avoid the ES6 destructuring? Whatever we decide, I think we should be consistent in this snippet and right now the rest is mostly ES3-style.

Reply:
Yessir